packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -605,6 +605,11 @@ class AnimationController extends Animation<double>
   /// repeat. The [status] will be reported as [AnimationStatus.reverse] when
   /// the animation runs from [max] to [min].
   ///
+  /// The animation repeats after a [period] duration. The [period] cannot be null 
```

Comment:
Maybe better:

Each run of the animation will have a duration of `period`. If `period` is not provided, [duration] will be used instead, which has to be set before [repeat] is called (either in the constructor or later by using the [duration] setter).

Reply:
ok updating witht his one.